src/config.rs

```diff
@@ -56,25 +57,23 @@ pub fn toggle_autostart() {
         config_path.push("grout");
         config_path.push("config.yml");
 
-        if let Ok(mut config) = fs::File::open(&config_path) {
+        if let Ok(mut config) = File::open(&config_path) {
             let mut config_str = String::new();
 
-            let _ = config.read_to_string(&mut config_str);
+            config
+                .read_to_string(&mut config_str)
+                .expect("Can not read config.");
```

Comment:
My point was, that reading and writing to the config should not just silently fail. Panicking in this case might not be the right call but the user should definitely know that an error has happened. My first instinct was to return `Result` and propagate the error further with `?`. However this would have required a more elaborated refactor of the functions in question.

Reply:
`anyhow` basically works like `Box<dyn Error...` so you can propagate across most error types in the same function. So it'll be easy to bubble it all up, then throw a end user friendly notification if that function fails, with the underlying error message.